src/modules/cpu/kernel/fog.hpp

```diff
@@ -79,6 +81,11 @@ RppStatus fog_u8_u8_host_tensor(Rpp8u *srcPtr,
         Rpp32u vectorIncrementPerChannel = 16;
 #if __AVX2__
         __m256 pIntensity = _mm256_set1_ps(intensityValue);
+        __m256 pGrayFactor = _mm256_set1_ps(grayValue);
+        __m256 pConversionFactor[3];
+        pConversionFactor[0] = _mm256_set1_ps(0.299f);
+        pConversionFactor[1] = _mm256_set1_ps(0.587f);
+        pConversionFactor[2] = _mm256_set1_ps(0.114f);
```

Comment:
In rppdefs.h, perhaps do a # define for a R_TO_GRAY_FACTOR, G_TO_GRAY_FACTOR and B_TO_GRAY_FACTOR. Use it in HOST, HIP - everywhere you need these 3 numbers.

Reply:
We seem to already have it but not using it - https://github.com/ROCm/rpp/blob/6cc67130cd7cc15f66eac16c9d5da294a8433fc1/src/include/hip/rpp_hip_common.hpp#L132-L134

Either define in hip common and cpu common separately, or just a single definition in rppdefs.h

Reply:
Although not part of this PR, we should ideally be using those # defines for this line too -> https://github.com/ROCm/rpp/blob/6cc67130cd7cc15f66eac16c9d5da294a8433fc1/src/include/cpu/rpp_cpu_common.hpp#L3222